manifests/components/kibana.jsonnet

```diff
@@ -21,7 +21,7 @@ local kube = import "../lib/kube.libsonnet";
 local kubecfg = import "kubecfg.libsonnet";
 local utils = import "../lib/utils.libsonnet";
 
-local KIBANA_IMAGE = "bitnami/kibana:5.6.12-r15";
+local KIBANA_IMAGE = "bitnami/kibana:5.6.13-r20";
```

Comment:
I think the safest is to keep Elasticsearch and Kibana versions synchronized. So I would bump Elasticsearch to 5.6.13 too.

Reply:
Indeed.  I believe Kibana logs a warning if the versions don't match (even if the mismatch is minor and safe, like this example presumably is).

Reply:
Bumping ElasticSearch image too so it's synchronized with Kibana